src/dlgProfilePreferences.cpp

```diff
@@ -1779,3 +1804,37 @@ void dlgProfilePreferences::slot_script_selected(int index)
         preview->setText(mpHost->getActionUnit()->getAction(itemId)->getScript());
     }
 }
+
+/*!
+ * \brief dlgProfilePreferences::slot_changeShowSpacesAndTabs
```

Comment:
Well you noted that using the second argument to `QObject::tr` i.e. the "disambiguation" string is now deprecated - and Qt instead wants users to give such stuff as specially marked comments but the documentation does not go out of its way to actually say that - I suspected that it was part of the ___QDoc___ system so I thought "what the heck" and starting using that {I've never used an inline code documentation system before so it is a bit of a learning curve/cliff for me}.

\<aside>As it happens I subsequently realised that the edbee library is using DOxygen and I got some useful guidance (or rather a large number of HTML documents) on the structure of that by running `doxygen` in the base of _that_ module...\</aside>